### ISSUE TITLE
Issue running npm run build or dev

### ISSUE THREAD

**Issue by Doudou8** [Samuel-Martineau/generator-svelte#3]
When I run `npm run build` or `npm run dev`, have this error:
```
[!] Error: ENOENT: no such file or directory, open '/Users/myuser/Desktop/notes/public/bundle2.css'
```
However, when I create a public folder first before running one of the two commands, the error doesn't occur. I think this means there is is something wrong with the folder creation.

**Comment by Samuel-Martineau** [comment]
Yeah, I plan to do a full rewrite of this project as soon as school ends (in 2 weeks)

**Comment by Samuel-Martineau** [comment]
Fixed in [2.0.0](https://www.npmjs.com/package/generator-svelte/v/2.0.0)
